packages/control-property-editor/src/components/ChangeIndicator.tsx

```diff
@@ -18,6 +18,7 @@ export function ChangeIndicator(props: ChangeIndicatorProps): ReactElement {
     const { saved, pending, id } = props;
     const rest = { id };
     const color = 'var(--vscode-terminal-ansiGreen)';
+    const type = id == 'sapUshellDashboardPage--ChangeIndicator' ? 'control' : 'property';
```

Comment:
@emrecevik06 remove this conditional from here. Extend this interface https://github.com/SAP/open-ux-tools/blob/main/packages/control-property-editor/src/components/ChangeIndicator.tsx#L5
and add pass type information where `ChangeIndicator` is consumed. e.g https://github.com/SAP/open-ux-tools/blob/main/packages/control-property-editor/src/panels/outline/Tree.tsx#L292